cmd/internal/storage/influxdb/influxdb_test.go

```diff
@@ -359,8 +361,28 @@ func createTestStats() (*info.ContainerInfo, *info.ContainerStats) {
 			"1GB": {Usage: 1234, MaxUsage: 5678, Failcnt: 9},
 			"2GB": {Usage: 9876, MaxUsage: 5432, Failcnt: 1},
 		},
-		ReferencedMemory: 12345,
-		PerfStats:        []info.PerfStat{{Cpu: 1, Name: "cycles", ScalingRatio: 1.5, Value: 4589}},
+		SmapsMemory: map[string]uint64{
+			"AnonHugePages":   1,
+			"Anonymous":       2,
+			"KernelPageSize":  3,
+			"LazyFree":        4,
+			"Locked":          5,
+			"MMUPageSize":     6,
+			"Private_Clean":   7,
+			"Private_Dirty":   8,
+			"Private_Hugetlb": 9,
+			"Pss":             10,
+			"Referenced":      11,
```

Comment:
Removing `ContainerStats.ReferencedMemory` breaks backward compatibility. I'm afraid we will have to keep it, for a while at least.

Reply:
It might even be reasonable to exclude it from the smaps metrics, or keep it duplicate forever. From what I understand, all other smaps metrics do not get reset using this referenced-cycle-count-reset-logic, but I'll have to investigate further.

Adding the referenced metric again is a matter of minutes and adjusting some tests. I think some early feedback on the general approach (and matters like compatiblity) is very important though.